create-dataset.sh

```diff
@@ -12,7 +12,6 @@ cd $OUTDIR
 ../create-tiles.sh ./SRTM_NE_250m_TIF/SRTM_NE_250m.tif 10 10
 ../create-tiles.sh ./SRTM_SE_250m_TIF/SRTM_SE_250m.tif 10 10
 ../create-tiles.sh ./SRTM_W_250m_TIF/SRTM_W_250m.tif 10 20
```

Comment:
Since you used `unar -D` there is no directory created after the archive being extracted.
```
../create-tiles.sh ./SRTM_NE_250m.tif 10 10
../create-tiles.sh ./SRTM_SE_250m.tif 10 10
../create-tiles.sh ./SRTM_W_250m.tif 10 20
```